packages/reactive-dict/reactive-dict.js

```diff
@@ -36,25 +36,34 @@ ReactiveDict = function (dictName) {
 };
 
 _.extend(ReactiveDict.prototype, {
-  set: function (key, value) {
+  // set() began as a key/value method, but we are now overloading it
+  // to take an object of key/value pairs, similar to backbone
+  // http://backbonejs.org/#Model-set
+
+  set: function (key_or_object, value) {
     var self = this;
 
+    if ((typeof key_or_object === 'object') && (value === undefined)) {
+      self._setObject(key_or_object);
+      return;
+    }
+
     value = stringify(value);
 
     var oldSerializedValue = 'undefined';
-    if (_.has(self.keys, key)) oldSerializedValue = self.keys[key];
+    if (_.has(self.keys, key_or_object)) oldSerializedValue = self.keys[key_or_object];
```

Comment:
I would add a `var key = key_or_object` once we have established we are dealing with a string key, so that the code below can remain unchanged.